lib/mutant/mutator/node/regexp/character_type.rb

```diff
@@ -0,0 +1,30 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Character type mutator
+        class CharacterType < Node
+          extend Mutant::AST::Sexp
+
+          map = {
+            s(:regexp_word_type) => s(:regexp_nonword_type),
+            s(:regexp_digit_type) => s(:regexp_nondigit_type),
+            s(:regexp_space_type) => s(:regexp_nonspace_type),
+            s(:regexp_word_boundary_anchor) => s(:regexp_nonword_boundary_anchor)
+          }
```

Comment:
What if we only map type to types and apply `s` in a map? This would reduce literal `s` method calls.